plugins/woocommerce/includes/admin/views/html-admin-dashboard-setup.php

```diff
@@ -27,3 +27,27 @@
 	</div>
 	<div class="clear"></div>
 </div>
+
+<script type="text/javascript">
+	/*global jQuery */
+	(function( $ ) {
+		const widget = $( '.dashboard-widget-finish-setup' );
+		const currentStep = widget.data( 'current-step' );
+		const totalSteps = widget.data( 'total-steps' );
+
+		$( document ).on( 'ready', function() {
```

Comment:
this throws a warning because `$( document ).on( 'ready'`  is deprecated in jQuery 3, and will works only if jQuery migrate is loaded. Anyway can safely be replaced with "load" that solves this issue

https://api.jquery.com/ready/

> There is also $(document).on( "ready", handler ), deprecated as of jQuery 1.8 and removed in jQuery 3.0. Note that if the DOM becomes ready before this event is attached, the handler will not be executed.